src/terminologySectionPanel/codesSection/codesSubSection/savedCodesSubSection/SavedCodesSubSection.tsx

```diff
@@ -136,6 +147,31 @@ export default function SavedCodesSubSection({ measureStoreCql, canEdit }) {
     }
   }, [measureStoreCql]);
 
+  const getCodeVersion = (
+    code,
+    parsedCodeSystem,
+    oid,
+    codeSystemMap,
+    matchedCodeSystemVersion
+  ) => {
+    if (codeSystemMap && !matchedCodeSystemVersion) {
+      //if version is added through UI, then check inn the cql meta data
```

Comment:
Minor spelling error: 
```suggestion
      //if version is added through UI, then check in the cql meta data
```